### PR TITLE
fix column name for do_dist + cmdscale output

### PR DESCRIPTION
**Pull request by double-y**
### Description

Fixed this issue

image

image

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [x] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/stats_wrapper.R [hunk 1]
```diff
@@ -181,7 +181,19 @@ do_cmdscale_ <- function(df,
   loadNamespace("tidyr")
   grouped_col <- grouped_by(df)
 
-  name_col <- avoid_conflict(grouped_col, "name")
+  # if pair1_col and pair2_col are with the same name
+  # like aaa.x and aaa.y (output of skv of do_dist),
+  # it (in this case, "aaa") sould be used as name column
```
Comment (hideaki) [inline R/stats_wrapper.R]: typo: sould

File: R/pairwise.R [hunk 2]
```diff
@@ -161,7 +161,11 @@ do_dist.kv_ <- function(df,
     rownames(ret) <- NULL
     if (!is.null(cmdscale_k)) {
       ret <- do_cmdscale_(ret, cnames[[1]], cnames[[2]], cnames[[3]], k = cmdscale_k)
+      # the label for each points should be the subject
```
Comment (hideaki) [inline R/pairwise.R]: Should be "each point".

File: R/stats_wrapper.R (hunk 1 above)
Comment (double-y) [inline R/stats_wrapper.R]: Right, fixed it

File: R/pairwise.R (hunk 2 above)
Comment (double-y) [inline R/pairwise.R]: Right, fixed it

**Pull request merged**